app/src/main/java/com/github/gotify/NotificationSupport.java

```diff
@@ -38,70 +47,19 @@ public static void createChannels(NotificationManager notificationManager) {
                             NotificationManager.IMPORTANCE_LOW);
             foreground.setShowBadge(false);
 
-            NotificationChannel messagesImportanceMin =
-                    new NotificationChannel(
-                            Channel.MESSAGES_IMPORTANCE_MIN,
-                            "Min priority messages (<1)",
-                            NotificationManager.IMPORTANCE_MIN);
-
-            NotificationChannel messagesImportanceLow =
-                    new NotificationChannel(
-                            Channel.MESSAGES_IMPORTANCE_LOW,
-                            "Low priority messages (1-3)",
-                            NotificationManager.IMPORTANCE_LOW);
-
-            NotificationChannel messagesImportanceDefault =
-                    new NotificationChannel(
-                            Channel.MESSAGES_IMPORTANCE_DEFAULT,
-                            "Normal priority messages (4-7)",
-                            NotificationManager.IMPORTANCE_DEFAULT);
-            messagesImportanceDefault.enableLights(true);
-            messagesImportanceDefault.setLightColor(Color.CYAN);
-            messagesImportanceDefault.enableVibration(true);
-
             NotificationChannel messagesImportanceHigh =
                     new NotificationChannel(
                             Channel.MESSAGES_IMPORTANCE_HIGH,
-                            "High priority messages (>7)",
+                            "Gotify notifications",
                             NotificationManager.IMPORTANCE_HIGH);
             messagesImportanceHigh.enableLights(true);
             messagesImportanceHigh.setLightColor(Color.CYAN);
             messagesImportanceHigh.enableVibration(true);
 
             notificationManager.createNotificationChannel(foreground);
-            notificationManager.createNotificationChannel(messagesImportanceMin);
-            notificationManager.createNotificationChannel(messagesImportanceLow);
-            notificationManager.createNotificationChannel(messagesImportanceDefault);
             notificationManager.createNotificationChannel(messagesImportanceHigh);
         } catch (Exception e) {
             Log.e("Could not create channel", e);
         }
     }
-
-    /**
-     * Map {@link com.github.gotify.client.model.Message#getPriority() Gotify message priorities to
-     * Android channels.
-     *
-     * <pre>
-     * Gotify Priority  | Android Importance
-     * <= 0             | min
-     * 1-3              | low
-     * 4-7              | default
-     * >= 8             | high
-     * </pre>
-     *
-     * @param priority the Gotify priority to convert to a notification channel as a long.
-     * @return the identifier of the notification channel as a String.
-     */
-    public static String convertPriorityToChannel(long priority) {
```

Comment:
The priority now gets ignored completely, I don't really like this cause we basically removing this feature without any replacement. I dunno if it is possible to still use the priority on the notification because I think the NotificationChannel takes precedence. @eternal-flame-AD what do you think about this?

Reply:
I don't think it would be possible. I think the only option is to create a channel for each possible notification importance level. Maybe we can create a channel group for each app and each has 3 channels inside for the 3 different importance levels?

Reply:
Isn't that a bit over the top having 3 channels for each app?

Users could just add a second app and set up channel options according to their needs or add one app which receives all messages with medium importance etc